gorums.proto

```diff
@@ -14,6 +14,7 @@ extend google.protobuf.MethodOptions {
   bool qc_future = 50004;
   bool qf_with_req = 50005;
   bool per_node_arg = 50006;
+  bool strict_ordering = 50007;
```

Comment:
What if we used: `ordered` as the option name. Maybe this would also make sense elsewhere in the code, but maybe not everywhere.